terms/diophantine_set.md

```diff
@@ -0,0 +1,17 @@
+# Diophantine Set
+## Definition
+
+A set $S \in Z^k$ is called ==Diophantine== if and only if there exists a integer coefficient multivariate polynomial $R_S$ such that
+$\mu \in S \Leftrightarrow \exists \omega \in Z^{k'} s.t. R_S(\mu, \omega)=0$, i.e $\mu \in S$  if $\mu$ is a part of a solution for a
+fixed Diophantine equation. We call $R_S$ as a ==representing polynomial== of $S$, and $\omega$ as an auxiliary witness. 
```

Comment:
Is it a display error?

Reply:
see [this](https://github.com/sota-zk-lab/zkp-documents/pull/39#discussion_r1524342816)

Reply:
see [this](https://github.com/sota-zk-lab/zkp-documents/pull/39#discussion_r1524342816)